admin/future_grants/V1.13.0__synapse_aggregate_future_tables.sql

```diff
@@ -0,0 +1,32 @@
+-- SYNAPSE_DATA_WAREHOUSE_DEV
+-- admin/ownership
```

Comment:
you should also be granting OWNERSHIP of future dynamic tables to <DOMAIN>_PROXY_ADMIN right?

<img width="1174" alt="image" src="https://github.com/user-attachments/assets/7435fb0e-5c97-466b-95eb-78584dfb3830" />